dev/integration_tests/web/analysis_options.yaml

```diff
@@ -0,0 +1,5 @@
+# Exclude lib/main.dart to avoid warnings about the non_nullable
+# experiment before it is enabled.
+analyzer:
```

Comment:
This is one of the changes: exclude this file specifically from analysis. The risk is low since we still need to run the file for a test later